io/src/main/java/io/smallrye/common/io/Messages.java

```diff
@@ -0,0 +1,27 @@
+package io.smallrye.common.io;
+
+import static java.lang.invoke.MethodHandles.lookup;
+
+import java.nio.file.FileSystem;
+import java.nio.file.Path;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "SRCOM", length = 5)
+interface Messages extends BasicLogger {
+    Messages log = org.jboss.logging.Messages.getBundle(lookup(), Messages.class);
```

Comment:
Shouldn't this be a Message Logger now that it extends `BasicLogger`? 

```suggestion
    Messages log = org.jboss.logging.Logger.getMessageLogger(lookup(), Messages.class, Messages.class.getPackageName());
```

Reply:
The lack of tests really shows, eh? :)